lib/scala/project-manager/src/main/scala/org/enso/projectmanager/service/ProjectServiceFailure.scala

```diff
@@ -102,4 +102,9 @@ object ProjectServiceFailure {
     */
   case class ComponentRepositoryAccessFailure(msg: String)
       extends ProjectServiceFailure
+
+  /** Signals about a failure in the configuration service. */
```

Comment:
```suggestion
  /** Signals a failure in the configuration service. */
```

This is a very minor thing but I think we usually write error descriptions in the form `Signals that X` or `Signals a failure of X` etc.